src/Framework/App.Ref/src/Microsoft.AspNetCore.App.Ref.sfxproj

```diff
@@ -0,0 +1,186 @@
+      <_TransportRoslynVersionedAnalyzers UnversionedAnalyzerPath="$([System.Text.RegularExpressions.Regex]::Replace('%(RecursiveDir)', '$(RoslynVersionPattern)', ''))" />
+      <FilesToPackage Include="@(_TransportRoslynVersionedAnalyzers)"
+                      ExcludeFromValidation="true"
+                      TargetPath="$(AnalyzersPackagePath)dotnet/%(_TransportRoslynVersionedAnalyzers.UnversionedAnalyzerPath)" />
+    </ItemGroup>
+
+    <!-- Package the analyzers and other Roslyn components produced in this repo. -->
+    <ItemGroup>
+      <FilesToPackage Include="@(AspNetCoreAnalyzer)"
+                      ExcludeFromValidation="true"
+                      TargetPath="$(AnalyzersPackagePath)dotnet/cs/" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CreatePackageOverrides"
+          DependsOnTargets="ResolveReferences"
+          Condition="'$(IsServicingBuild)' != 'true'">
+    <ItemGroup>
+      <!-- Use package version for non-Runtime references. -->
+      <_AspNetCoreAppPackageOverrides Include="@(ReferencePath->'%(NuGetPackageId)|%(NuGetPackageVersion)')"
+          Condition="!Exists('$(RuntimeTransportReferenceDirectory)%(ReferencePath.NuGetPackageId).dll') AND
+            '%(ReferencePath.NuGetPackageId)' != 'Microsoft.NETCore.App' AND
+            '%(ReferencePath.NuGetPackageId)' != 'Microsoft.Internal.Runtime.AspNetCore.Transport' AND
+            '%(ReferencePath.NuGetSourceType)' == 'Package' " />
+
+      <!--
+        Use NETCore.App version for Runtime references.
+        This is the same as above except in "stabilized" version builds, where the transport package version
+        would be a non-shipping unstablized version, but the NETCore.App version is the shipping version.
+      -->
+      <_AspNetCoreAppPackageOverrides Include="@(ReferencePath->'%(FileName)|$(MicrosoftNETCoreAppRuntimeVersion)')"
+      Condition="'%(ReferencePath.NuGetPackageId)' == 'Microsoft.Internal.Runtime.AspNetCore.Transport' AND
+                 '%(ReferencePath.NuGetSourceType)' == 'Package' " />
+
+      <!-- Use shared framework version for project references. -->
+      <_AspNetCoreAppPackageOverrides Include="@(ReferencePath->'%(FileName)|$(ReferencePackSharedFxVersion)')"
+          Condition=" '%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference' " />
+    </ItemGroup>
+
+    <WriteLinesToFile
+        Lines="@(_AspNetCoreAppPackageOverrides)"
+        File="$(ReferencePackageOverridesPath)"
+        Overwrite="true" />
+  </Target>
+
+  <Target Name="PublishToSharedLayoutRoot" BeforeTargets="Build">
+    <MSBuild Projects="$(MSBuildProjectFullPath)"
+             Targets="PublishToDisk"
+             Properties="OutputPath=$(TargetingPackLayoutRoot)" />
+  </Target>
+
+  <Target Name="AddLinuxPackageInformation" BeforeTargets="GetDebInstallerJsonProperties;GetRpmInstallerJsonProperties">
```

Comment:
Can you add a comment about what this target is for?

Reply:
Will do